lib/kube/proxy/auth.go

```diff
@@ -130,6 +131,7 @@ func getKubeCreds(ctx context.Context, log logrus.FieldLogger, tpClusterName, ku
 	res := make(map[string]*kubeCreds, len(cfg.Contexts))
 	// Convert kubeconfig contexts into kubeCreds.
 	for cluster, clientCfg := range cfg.Contexts {
+		clientCfg.TLSClientConfig.Insecure = lib.IsInsecureDevMode()
```

Comment:
Is this related to EC2 tags or fixing some other issue?

Reply:
It isn't needed for the change itself, but it is for the test. The test cluster created by the `enableKubernetesService()` helper won't work without either doing this or setting up TLS properly.